packages/reqresp/test/unit/response/index.test.ts

```diff
@@ -22,8 +23,9 @@ const testCases: {
     id: "Yield two chunks, then throw",
     protocol: pingProtocol(async function* () {
       const payload = sszSnappyPing.binaryPayload;
-      yield {...payload, boundary: {fork: payload.fork}};
-      yield {...payload, boundary: {fork: payload.fork}};
+      const boundary = {fork: payload.fork, epoch: config.forks[payload.fork].epoch};
```

Comment:
Probably more consistent to do this

```suggestion
      const epoch = config.forks[payload.fork].epoch;
      const boundary = config.getForkBoundaryAtEpoch(epoch);
```

Reply:
yeah seems more readable